ncm-openvpn/src/main/perl/openvpn.pm

```diff
@@ -115,7 +115,7 @@ sub Configure
 
     if ($changed) {
 	$self->verbose("Restarting OpenVPN daemon");
-	CAF::Process->new([qw(/etc/init.d/openvpn restart)],
+	CAF::Process->new([qw(service openvpn restart)],
```

Comment:
The real fix here is to move to CAF::Service which handles systemd natively, but as a quick fix I don't have a problem with this. I think an absolute path might be a better idea rather than relying on $PATH though?